resources/views/feeds/blog.edge

```diff
@@ -16,11 +16,11 @@
     @each(post in blogPosts)
       <item>
         <title><![CDATA[{{ post.title }}]]></title>
-        <link>{{ route('blog.show', [post.slug]) }}</link>
-        <guid>{{ post.slug }}</guid>
+        <link>https://adonisjs.com{{ route('blog.show', [post.slug]) }}</link>
+        <guid>https://adonisjs.com{{ route('blog.show', [post.slug]) }}</guid>
         <pubDate>{{ post.publishedAt }}</pubDate>
-        <description><![CDATA[ {{{ post.contentHtml.summary }}} ]]></description>
-        <content:encoded><![CDATA[ {{{ post.contentHtml.contents }}} ]]></content:encoded>
+        <description><![CDATA[ {{{ sanitize(post.contentHtml.summary) }}} ]]></description>
```

Comment:
Sanitization is necessary so that we don't include dangerous elements in the output, such as script tags, etc. This solves almost all of the content related warnings in the validation tool.